src/scythe-bidder/create-room.tsx

```diff
@@ -2,31 +2,91 @@
 
 import React from "react";
 import { jsx } from "@emotion/core";
-import { Button, Card, Form, notification, Select, Switch } from "antd";
+import { Button, Card, Form, notification, Select, Modal } from "antd";
 import client from "./client";
-import { FACTIONS_BASE, FACTIONS_IFA, MATS_BASE, MATS_IFA } from "./constants";
+import {
+  FACTIONS_BASE,
+  FACTIONS_IFA,
+  FACTIONS_HI,
+  FACTIONS_LO,
+  MATS_BASE,
+  MATS_IFA,
+  MATS_HI,
+  MATS_LO,
+} from "./constants";
 import { MAX_PLAYERS_BASE, MAX_PLAYERS_IFA, MIN_PLAYERS } from "./constants";
 import { SCYTHE_BIDDER } from "./constants";
+import { QuestionCircleFilled } from "@ant-design/icons";
 
 export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
   const [numPlayers, setNumPlayers] = React.useState(2);
   const [isIfaActive, setIsIfaActive] = React.useState<boolean>(true);
+  const [activeCombinations, setActiveCombinations] = React.useState<string>(
+    "IFA"
+  );
   const maxPlayers = isIfaActive ? MAX_PLAYERS_IFA : MAX_PLAYERS_BASE;
 
+  const { Option } = Select;
+
+  const gameSettingLabel = [
+    <div>
+      Game setting <QuestionCircleFilled onClick={settingInformation} />
+    </div>,
+  ];
+
+  function settingInformation() {
+    Modal.info({
+      title: "Game setting details",
+      content: (
+        <div css={{ marginTop: 24 }}>
+          <p>IFA adds Togawa, Albion, Innovative, and Militant.</p>
+          <p>Base includes the five default faction and mat options.</p>
+          <p>
+            Hi-tier setting removes Albion, Togawa, Agricultural, and
+            Engineering
```

Comment:
nit: I think this should end with a period.

Reply:
Maybe it should also specify that it removes those from the IFA setting to be clear that Militant and Innovative are in.